CSC8503/Main.cpp

```diff
@@ -31,24 +31,34 @@ using namespace CSC8503;
 #include <sstream>
 #include "DebugNetworkedGame.h"
 
+namespace{
+    constexpr int SERVER_CHOICE = 1;
+}
+
 int main(){
-    Window* w = Window::CreateGameWindow("CSC8503 Game technology!", 1280, 720);
+    Window* w = Window::CreateGameWindow("CSC8503 Game technology!", 800, 600, false);
```

Comment:
I knew there was a reason I got the screen size as a variable, rather than hard coded it! :D